cosmos/dbt/graph.py

```diff
@@ -424,18 +428,21 @@ def load_via_dbt_ls_without_cache(self) -> None:
             raise CosmosLoadDbtException("Unable to load project via dbt ls without a profile config.")
 
         with tempfile.TemporaryDirectory() as tmpdir:
-            logger.debug(
-                f"Content of the dbt project dir {self.render_config.project_path}: `{os.listdir(self.render_config.project_path)}`"
-            )
+            logger.debug(f"Content of the dbt project dir {project_path}: `{os.listdir(project_path)}`")
             tmpdir_path = Path(tmpdir)
 
             create_symlinks(project_path, tmpdir_path, self.render_config.dbt_deps)
 
-            if self.project.partial_parse and self.cache_dir:
-                latest_partial_parse = cache._get_latest_partial_parse(project_path, self.cache_dir)
+            latest_partial_parse = None
+            if self.project.partial_parse:
+                if self.should_use_partial_parse_cache() and self.cache_dir:
+                    latest_partial_parse = cache._get_latest_partial_parse(project_path, self.cache_dir)
+                else:
+                    latest_partial_parse = get_partial_parse_path(project_path)
+
+            if latest_partial_parse is not None and latest_partial_parse.exists():
                 logger.info("Partial parse is enabled and the latest partial parse file is %s", latest_partial_parse)
-                if latest_partial_parse is not None:
-                    cache._copy_partial_parse_to_project(latest_partial_parse, tmpdir_path)
+                cache._copy_partial_parse_to_project(latest_partial_parse, tmpdir_path)
```

Comment:
I am wondering if we need another check here to see if cache is enabled and only then copy it?

Reply:
This line needs to be run regardless of caching - this how we're fixing the original issue